include/fastdds/statistics/topic_names.hpp

```diff
@@ -16,54 +16,54 @@
  * @file topic_names.hpp
  */
 
-#ifndef _STATISTICS_TOPIC_NAMES_HPP_
-#define _STATISTICS_TOPIC_NAMES_HPP_
+#ifndef _FASTDDS_STATISTICS_TOPIC_NAMES_HPP_
+#define _FASTDDS_STATISTICS_TOPIC_NAMES_HPP_
 
 namespace eprosima {
 namespace fastdds {
-namespace dds {
 namespace statistics {
+namespace dds {
```

Comment:
I think these constants should be on the `eprosima::fastdds::statistics` namespace (i.e. not `::dds`)

Reply:
Done in 5a672028fdeeea4ea622c9f20f9c8e5e6b741f78